components/questionCard.js

```diff
@@ -52,14 +52,48 @@ const StyledField = styled.input`
   overflow: auto;
   border: none;
   font-family: 'HK Grotesk';
-  cursor: none;
```

Comment:
Lgtm, just one minor thing, can you add "user-select: none;"?

When I hover over the text box for the question title, the cursor changes into an edit cursor, so a user may think they can edit. Can we make it so that the cursor remains the same upon hover?